benchmarks/graphics_pipeline/GraphicsBenchmarkApp.h

```diff
@@ -254,6 +247,20 @@ class GraphicsBenchmarkApp
     std::array<grfx::PipelineInterfacePtr, kFullscreenQuadsTypes.size()> mQuadsPipelineInterfaces;
     std::array<grfx::ShaderModulePtr, kFullscreenQuadsTypes.size()>      mQuadsPs;
 
+    // Metrics Data
+    struct MetricsData
+    {
+        enum EMetricsTypes : size_t
```

Comment:
Is that "E" in front of "MetricTypes" for "Enum"? Is that a naming convention?

Reply:
yes, but not consistent with other enum naming in BW, removed E